.php_cs.dist

```diff
@@ -17,25 +17,27 @@ $finder = (new PhpCsFixer\Finder())
     ->in(__DIR__.'/views')
 ;
 
+/**
+ * @version ^2.0
```

Comment:
@alcohol Is this valid? :smile:

Reply:
I would not use @version since that is sort of "official".

Can I propose:

```
/* This config requires php-cs-fixer version ^2.3 */
```

Reply:
Or whichever version we require :-)

Reply:
As we support `PHP 7`, we can support PHPCS version `^2.9` :smile: